Add 2023-11-08 agenda and minutes (#595)
solid/specification: meetings/2023-11-08.md

* Join queue to talk.
* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.

### Participation and Code of Conduct
* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/).
* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
* If this is your first time, welcome! please introduce yourself.

### Scribes
* Aaron Coburn

### Introductions
* Eric Jahn: My main perspective is the need for consent sharing. That would be patient or human services client. 

> * Vivian: I’m interested in interoperability on the web, and I think RDF is the way to go.

Reviewer: These look like they've already happened, but this meeting doesn't happen for ~21.5 hours (as of this typing)....

I might suggest tweaking this template, so it distinguishes between boilerplate (that doesn't change, and doesn't need to repeat, from meeteing to meeting); structure; actual agenda; and meeting notes/minutes.

It *should* be possible to line each session's agenda up against the minutes for that session, and thereby see what was planned to be discussed Vs. what was actually discussed — most importantly revealing what was *not* discussed, so that cycles onto the next agenda (unless an explicit decision is made to discard some topic(s)).

Author: My bad. The meeting hasn't happened. That text is from the previous meeting. I've removed it in https://github.com/solid/specification/pull/595/commits/228c3157de171aae4125374d38a042e8ca526e70

* SC: Any objections or suggestions?
* SC: It is already a work item, we just didn't put it in a separate repo, I PRd myself as editor and added Henry as an author.
* eP: With HttpSig as Work Item, there were two parts. Mentioned signing http messages from .. IETF. The other part was about publishing keys. For example. GNAP has a few different ways of proof of possession. Perhaps have on spec for publishing keys, then signing can be different? For client authn, publicJwk, you can use the same way of discovery and publishing keys.
* SC: I need to do more homework on this, I would be interested in having a STM on it.
* SC: Regarding the PR, I want to acknowledge an honor Henry. I want to know if people are ok with what I proposed.
* AC: I am +1 on SC's change
* AC: It is important to recognize Henry's contributions. At the same time it is important to have a clear person who is responsible for moving the work item forward.
* SC: That was the idea to signal who is the contact person now.

### Remove contributors, add authors
URL: https://github.com/solid/web-access-control-spec/pull/126

* SC: Any objections or suggestions?

> * SC: There are only few specs which mention contributors, they usually have editors and authors.
> * SC: I asked Henry long time ago can I add him as author. He didn't give me a clear answer.
> * eP: IIRC, Henry was talking about WAC+ and had various functionality. Maybe he didn't want to be listed as author there was no consensus to include his changes so perhaps speculative but maybe he was talking about it. How ACP's stuff could also work in WAC re same use cases, evaluations .. https://github.com/solid/authorization-panel/tree/main/proposals/evaluation
> * SC: I talked with Matthieu and actually some things in ACP were improved based on Henry's feedback.

Reviewer: ```suggestion
* SC: There are only a few specs which mention contributors. They usually have editors and authors.
* SC: I asked Henry a long time ago if I could add him as an author. He didn't give me a clear answer.
* eP: IIRC, Henry was talking about WAC+ having various functionality. Maybe he didn't want to be listed as author. There was no consensus to include his changes, so perhaps speculative, but maybe he was talking about it. How ACP's stuff could also work in WAC re: same use cases, evaluations ... https://github.com/solid/authorization-panel/tree/main/proposals/evaluation
* SC: I talked with Matthieu, and actually some things in ACP were improved based on Henry's feedback.
```

### Solid CG Chair Election Procedure
URL: https://github.com/solid/specification/discussions/582

* SC: Notes: https://hackmd.io/a6jv3bfnQBGace7Ajd_ePw
* SC: I was talking with W3C ... We don't have complete support as it would be for TAG or AC elections.
* SC: We have some code that gets data from W3C API
* SC: https://github.com/VirginiaBalseiro/solid-ecosystem-monitor/pull/2

> * SC: The idea is that by default we take participant with lowest w3c id (which is incremental number)
> * SC: We will offer all affiliations to change a name.
> * SC: In the end we will give a list of eligible voters to W3C staff
> * AC: That sounds fine
> * SC: While you here maybe you can give heads up to Inrupt since it has 14 participants.
> * AC: I would probably be the inrupt name, you could use my name as default/designated Inrupt representative.
> * eP: I will add you as designated and link to the minutes.

Reviewer: ```suggestion
* SC: The idea is that, by default we take the participant with lowest W3C ID (which is incremental number)
* SC: We will offer all affiliations the chance to change their designee.
* SC: In the end, we will give a list of eligible voters to W3C staff
* AC: That sounds fine
* SC: While you are here, maybe you can give a heads-up to Inrupt, since it has 14 participants.
* AC: I would probably be the Inrupt name; you could use my name as default/designated Inrupt representative.
* eP: I will add you as designated, and link to the minutes.
```

* AC: Whatever is easiest for your tooling is fine.

### Align shared core terminology
URL: https://github.com/solid/specification/discussions/557

> * SC: The discussion served its purpose (but we can improve as suggested by Ted). We should considering convert this to issues and/or PRs. And try to cover them for the next releases of the specs.
> * SC: We need to be careful with what terminology we are changing. We could move that into small PRs and issues.
> * eP: We may also want to draw a balance and provide a documentation where they can be input for the WG. Not sure if PRs would be best way because we hope it'll be transitioned to WG.
> * SC: I think PR or any input we make matters. I hope that WG will need to review all the material. If there is a text out there I try to process it. PR is more about let's make a decision. The WG can always take a different direction.

Reviewer: ```suggestion
* SC: The discussion served its purpose (but we can improve as suggested by Ted). We should consider converting this to issues and/or PRs. And try to cover them for the next releases of the specs.
* SC: We need to be careful with the terminology we are changing. We could move that into small PRs and issues.
* eP: We may also want to draw a balance, and provide documentation where they can be input for the WG. Not sure if PRs would be best way, because we hope it'll be transitioned to WG.
* SC: I think PR or any input we make matters. I hope that WG will review all the material. If there is a text out there, I try to process it. PR is more about "let's make a decision." The WG can always take a different direction.
```

* SC: Will try to take and make sense of it. Instead of looking at discussions, WG can look at the draft or PR.

### i18n and n11n of resource identifiers
URL: https://github.com/solid/specification/pull/575

* SC: All, please review.
* SC: Didn't get a chance to review but very interested in us getting this (in one shape or another) into 0.11.0.

> * SC: My informal view is that there might be to much detail for the solid protocol spec. It might be almost overwhelming. I would expect that implementers would use existing libraries. There must be a conversation how much detail belongs in the solid protocol and how much in the existing specs.
> * AC: My feeling is to generally not recapitulate what is stated in existing specification. There is a complexity to that but having a slim specification which refers to others seems much better.

Reviewer: ```suggestion
* SC: My informal view is that there might be too much detail for the solid protocol spec. It might be almost overwhelming. I would expect that implementers would use existing libraries. There must be a conversation about how much detail belongs in the solid protocol, and how much in the existing specs.
* AC: My feeling is to generally not recapitulate what is stated in existing specification. There is a complexity to that, but having a slim specification which refers to others seems much better.
```

* SC: I agree

### Please review CG Report requirements
URL: https://github.com/solid/specification/issues/587

* SC: Concerns releasing CG-DRAFT/FINALs.
* SC: See also next topic re Solid Protocol v0.11.0

> * SC: There was some complains that our specs look like TRs.
> * SC: We need to release CG drafts and final reports. I don't think we need to do it if there are no actual req and editorial changes. So, lets wrap up pending issues/milestones for DRAFT/FINAL releases.

Reviewer: ```suggestion
* SC: There were some complaints that our specs look like TRs.
* SC: We need to release CG drafts and final reports. I don't think we need to do it if there are no actual req and editorial changes. So, let's wrap up pending issues/milestones for DRAFT/FINAL releases.
```
